src/foam/u2/dialog/Popup.js

```diff
@@ -71,7 +71,7 @@ foam.CLASS({
       top: 0;
     }
     ^inner {
-      height: auto;
+      height: 85vh;
```

Comment:
This should be in ApplicationPopup, dont want to add styles to popups that arent wizards since they will have different sizes